Makefile

```diff
@@ -125,6 +125,9 @@ check:
 	@echo "errcheck"
 	@! errcheck -ignore 'bytes:Write.*,io:(Close|Write),net:Close,net/http:(Close|Write),net/rpc:Close,os:Close,database/sql:Close,github.com/spf13/cobra:Usage' $(PKG) | grep -vE 'yacc\.go:'
 	@echo "vet"
+	@! go tool vet . 2>&1 | \
```

Comment:
That would be a little simpler, but I'd rather not have this line to differ from the `--shadow` version any more than necessary.